calico-enterprise/multicluster/federation/kubeconfig.mdx

```diff
@@ -430,10 +430,10 @@ calicoq eval "all()"
 If all remote clusters are accessible, calicoq returns something like the following. Note the remote cluster prefixes: there should be endpoints prefixed with the name of each RemoteClusterConfiguration in the local cluster.
 ```
 Endpoints matching selector all():
-  Workload endpoint remote-cluster-1/host-1/k8s/kube-system.kube-dns-5fbcb4d67b-h6686/eth0
```

Comment:
I don't think these replacements are valid, e.g. kube-dns is not deployed into calico-system

Reply:
I have reverted the changes, since calicoq eval all() can list the endpoint in kube-system namespace.
```
vara@vara:~/bzprofiles/Clusters/tc_std_fed_2calicoq eval "all()"|grep kube-system
  Workload endpoint vara-bz-d3ax-kadm-node-0/k8s/kube-system.coredns-7db6d8ff4d-6mjwf/eth0
  Workload endpoint cluster-a/vara-bz-6udx-kadm-node-0/k8s/kube-system.coredns-7db6d8ff4d-fmw9c/eth0
  Workload endpoint cluster-a/vara-bz-6udx-kadm-node-0/k8s/kube-system.coredns-7db6d8ff4d-mt822/eth0
  Workload endpoint vara-bz-d3ax-kadm-node-0/k8s/kube-system.coredns-7db6d8ff4d-ghg7l/eth0
```